src/lib/datepicker/datepicker.md

```diff
@@ -32,6 +32,17 @@ can easily be used as a prefix or suffix on the material input:
 </mat-form-field>
 ```
 
+If you want to customize the icon that is rendered inside the `mat-datepicker-toggle`, you can do so
+by using the `matDatepickerToggleIcon` directive:
+
+```html
+<input [matDatepicker]="myDatepicker">
+<mat-datepicker-toggle [for]="myDatepicker">
+  <img matDatepickerToggleIcon src="my-custom-icon.png">
+</mat-datepicker-toggle>
+<mat-datepicker #myDatepicker></mat-datepicker>
+```
```

Comment:
can you turn this into an embedded example?

Reply:
Done.